editor/js/ui/sidebar.js

```diff
@@ -30,7 +30,7 @@ RED.sidebar = (function() {
     function addTab(title,content,closeable) {
         $("#sidebar-content").append(content);
         $(content).hide();
-        sidebar_tabs.addTab({id:"tab-"+title,label:title,closeable:closeable});
+        sidebar_tabs.addTab({id:"tab-"+title,label:RED._("sidebar."+title),closeable:closeable});
```

Comment:
I'm setting up a fix for this now. I'll squash it into the commit and push to my master.